site/tryit/README.md

```diff
@@ -35,37 +37,49 @@ First annotate your docker bridge local network address. Usually it's `172.17.0.
 
 
 ::: warning
-NOTE: if you have an ssh server running locally you should stop it
+If you have an **ssh server** running locally you should **stop** it
 :::
 
 ```
 docker run -d --name gitea -v /path/to/gitea-data:/data -p 3000:3000 -p 22:22 gitea/gitea:latest
 ```
-
-This will save your gitea data inside your host `/path/to/gitea` directory
+where:
+* `-v /path/to/gitea-data:/data` will mount a local `/path/to/gitea-data` directory (be sure to change it) to `/data` inside the container
+   
 
 #### Setup gitea
 
 
-* Access gitea on `http://172.17.0.1:3000`
-* In the initial setup page you should change:
+1. Access gitea on `http://172.17.0.1:3000`
+
+2. In the initial setup page you should change:
    * the *Gitea Base URL* to `http://$YOURDOCKERLOCALIP:3000` (i.e. `http://172.17.0.1:3000`)
    * the *SSH Server Domain* to `$YOURDOCKERLOCALIP` (i.e. `172.17.0.1`)
-* Register a new user
-* Under your user settings, add your ssh public key (to be able to push to repositories)
+
+3. Register a new user
+
+4. Under your user settings, add your `ssh-public-key` (to be able to push to repositories)
+
+5. Now create an oauth2 app under your user settings -> Applications -> Manage OAuth2 Applications. As the applicatin *name* set "Agola" and as *redirect uri* `http://172.17.0.1:8000/oauth2/callback`. 
 
 ::: warning
-Your gitea image should be new enough to support oauth2 (if pull a newer image or skip the next step). 
+Keep note of the provided ***Client ID*** and ***Client Secret***.
 :::
 
-* Now create an oauth2 app under your user settings -> Applications -> Manage OAuth2 Applications. As the applicatin name set "Agola" and as redirect uri `http://172.17.0.1:8000/oauth2/callback`. Keep note of the provided "Client ID" and "Client Secret".
+::: warning
+Your gitea image should be new enough to support oauth2, if pull a newer image or skip the **next step** . 
+* Gitea Version >= 1.8.0  &ensp; - &ensp; Have *OAuth2* Authentication
+:::
```

Comment:
After a second read this doesn't sound really clear. I'll just leave a simpler warning like

```
::: warning
Your gitea image should be new enough to support oauth2 (gitea version >= 1.8.0) or just skip this step.
:::
```